db/migrate/20180507155745_create_rents.rb

```diff
@@ -0,0 +1,12 @@
+class CreateRents < ActiveRecord::Migration[5.1]
+  def change
+    create_table :rents do |t|
+      t.references :user, foreign_key: true
+      t.references :book, foreign_key: true
+      t.date :from
+      t.date :to
```

Comment:
All the fields are required, so neither of them could be null. Add `null: false` to each one.
As before, assume this already ran in a production server and create a new migration to add `null: false`, don't modify this one!